### ISSUE TITLE
Is this project active?

### ISSUE THREAD

**Issue by raguay** [coderhaoxin/handlebars#31]
Hi,

I want to use handlebars in my golang based server, but I haven't found a library that is complete. Is this one still being developed? When will it be usable with helper functions? Thanks.

**Comment by coderhaoxin** [comment]
Yes, it's active. I'll continue to develop this repo, but not anytime soon.
For `helper functions`, should be completed in early October.
